src/nvim/edit.c

```diff
@@ -3825,6 +3831,21 @@ static void ins_compl_insert(void)
     compl_used_match = FALSE;
   else
     compl_used_match = TRUE;
+
+  // Set completed item.
+  // { word, abbr, menu, kind, info }
+  dict_T *dict = dict_alloc();
+  dict_add_nr_str(dict, "word", 0L,
+      SHOULD_BE_STR(compl_shown_match->cp_str));
```

Comment:
All these arguments should be aligned according to the opening parenthesis after `dict_add_nr_str`. Alternatively, all arguments should be moved out of the line with the function name.